peerinst/templates/peerinst/collection/collection_detail.html

```diff
@@ -150,7 +151,7 @@ <h1 class="mdc-typography--display3">{% trans 'Collections' %}
               </div>
               <div class="mdc-card__action-icons">
                 <div class="mdc-card__action-buttons">
-                  <div class="mdc-typography--caption">
+                  <div class="mdc-typography--caption" id="caption">
```

Comment:
id must be unique in the DOM: here you will have a bunch of elements with the same id.  Add "caption" to the class list instead.